raster-foundry/raster-foundry
# Add third party credentials to organizations

## Overview

This PR:
  * adds new columns for third party credentials to `organizations` table
  * populates credential fields in `users` table using the ones in the organization the user belongs to upon login, if there is no such credentials in `users` table

### Checklist

- ~Styleguide updated, if necessary~
- ~[Swagger specification](https://github.com/raster-foundry/raster-foundry-api-spec) updated~
- ~Any content changes are properly templated using `BUILDCONFIG.APP_NAME`~
-  ~Any new SQL strings have tests~


## Testing Instructions

 * Run migrations
 * Run the following in a sql shell:
```sql
update organizations set dropbox_credential = 'asdf' where id = 'dfac6307-b5ef-43f7-beda-b9f208bb7726';
update organizations set planet_credential = 'zxcv' where id = 'dfac6307-b5ef-43f7-beda-b9f208bb7726';
```
 * Login as a dev user
 * Check `dropbox_credential` and `planet_credential` fields in `users` table in a sql shell:
     - if you had such fields populated in the `users` table before, the values should stay as they were
     - if not, they should be populated by the values shown in the sql above

Closes #3386
